packages/libs/bootstrap-lib/docs/launch-post.md

```diff
@@ -0,0 +1,332 @@
+Title: Release @kadena/client
+
+We've created a library that allows Javascript/Typescript users to easily
+interact with the Kadena Blockchain. We've explicitly kept out creating
+contracts as it's way more complicated to transpile Javascript to Pact, as
+contracts need to be Pact lang in the end anyway.
+
+Interaction with the Kadena Blockchain works in multiple ways, and they're
+described in this article. We'll also go into the concepts and rationale of
+@kadena/client.
+
+- [Prerequisites](#prerequisites)
+- [Bootstrapping @kadena/client](#bootstrapping-kadenaclient)
+  - [Load contracts from the blockchain](#load-contracts-from-the-blockchain)
+  - [Manually singing the transaction](#manually-singing-the-transaction)
+  - [Automated sign request to **Chainweaver desktop**](#automated-sign-request-to-chainweaver-desktop)
+- [Building a simple transaction using a template](#building-a-simple-transaction-using-a-template)
+  - [Load the contract repository](#load-the-contract-repository)
+  - [Generate code from templates](#generate-code-from-templates)
+  - [A function is generated from a template](#a-function-is-generated-from-a-template)
+  - [Using the generated code](#using-the-generated-code)
+- [Further development](#further-development)
+- [Contact the team](#contact-the-team)
+
+# Prerequisites
+
+We're not going over installation of nodejs and package management, this
+information is widely available on the internet. We will provide the bare
+minimum of the things you need to do in order to start using @kadena/client:
+
+- install nodejs version 14.x or 16.x
+- create a directory, bootstrap a `package.json` by running `npm init` or
+  `npm init -y` to use defaults
+- install the client `npm install @kadena/client`
+- install the commandline tool `npm install @kadena/pactjs-cli`
```

Comment:
should we add `--save-dev`

Reply:
That's by default

Reply:
Good call actually, the pactjs cli could be incorporated in a prod workflow somehow and break if we suggest save-dev

Doesn't save much of anything imho anyway